components/standings/commons/standings_storage.lua

```diff
@@ -32,7 +32,11 @@ function StandingsStorage.run(data)
 		Opponent = Lua.import('Module:'.. data.opponentLibrary, {requireDevIfEnabled = true})
 	end
 
-	local standingsIndex = tonumber(data.standingsindex) or 0
+	local standingsIndex = tonumber(data.standingsindex)
+
+	if not standingsIndex then
+		error('No standingsindex specified')
+	end
```

Comment:
Can skip this conversion and check since this logic is already in both `.table` and `.entry`.